### PR TITLE
Update KafkaClient configuration and Docker to use correct port

### PR DESCRIPTION
**Pull request by mirpo**

### CODE REVIEW

File: src/KafkaUtil.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ module.exports = class KafkaUtil extends events.EventEmitter {
         super()
         this.dataTopic = dataTopic
         this.partitioner = partitioner
-        this.kafkaClient = kafkaClient || new kafka.KafkaClient(zookeeper, `streamr-kafka-producer-${Date.now()}`)
+        this.kafkaClient = kafkaClient || new kafka.KafkaClient({
+            kafkaHost: zookeeper
```
Comment (hpihkala) [inline src/KafkaUtil.js]: @mirpo this doesn't look right. You're passing the zookeeper address as the kafka host address. According to [the docs](https://www.npmjs.com/package/kafka-node) you should be passing one or more kafka node addresses to it.

Older versions of `kafka.KafkaClient` took the zookeeper address as the first argument. I wonder if this broke in a `kafka-node` version upgrade?

File: Dockerfile [hunk 2]
```diff
@@ -14,7 +14,7 @@ RUN apk add --update python build-base && npm install && apk del python build-ba
 EXPOSE 8890
 
 # Default environment variables
-ENV ZOOKEEPER_HOST zookeeper
+ENV ZOOKEEPER_HOST zookeeper:9092
```
Comment (hpihkala) [inline Dockerfile]: `9092` is not a zookeeper port. Are you trying to pass a kafka node address in the value `ZOOKEEPER_HOST`? You should probably rename the env variable.

**Comment by hpihkala** [comment]
Why merged without review?

**Comment by mirpo** [comment]
@hpihkala My fault, thought that it'll be a super fast hotfix. Created #49

**Pull request merged**